modules/vba_documents/spec/workers/flipper_status_alert_spec.rb

```diff
@@ -91,6 +91,7 @@
     end
 
     it 'does not notify Slack when all features are enabled' do
+      allow(YAML).to receive(:load_file).and_return({ 'common' => %w[feature1 feature2] })
```

Comment:
When I removed the last remaining feature from `enabled_features.yml`, the "no features found" Slack alert kicked in before the `BulkFeatureChecker` could be called, leading to test failures.